sql/moz-fx-data-shared-prod/analysis/unified_metrics_v1/query.sql

```diff
@@ -0,0 +1,258 @@
+WITH unioned_source AS (
+  SELECT
+    *,
```

Comment:
do we expect the underlying table to experience schema changes? Maybe it's better to explicitly list the columns for the union statement to avoid the query breaking when one of the tables gets an extra column? Unless we say we want to fail then.

Reply:
Is in a list now in the most recent version